src/middleware/sitemap/index.js

```diff
@@ -13,7 +13,7 @@ let packagesPromise = updatePackages();
 
 module.exports = async (ctx) => {
 	ctx.params.page = ctx.params.page.replace(/\.xml$/, '');
-	let pages = (await readDirRecursive(viewsPath + '/pages', [ '_*' ])).map(p => path.relative(viewsPath + '/pages', p).replace(/\\/g, '/').slice(0, -5));
+	let pages = (await readDirRecursive(viewsPath + '/pages', [ '_*' ])).map(p => path.relative(viewsPath + '/pages', p).replace(/\\/g, '/').slice(0, -7));
```

Comment:
Why 7?

Reply:
You are right, i misunderstood this moment, fixed back